feat: ES-9 add EDMM Model (K8s) of otel-shop

eliasmueller opened the pull request:
# Changes

Please provide a brief description of the changes here.

## Merge Requirements

For new features contributions please make sure you have completed the following
essential items:

* [ ] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs][]
* [ ] Appropriate Helm chart updates in the [helm-charts][]

<!--
A Pull Request that modifies instrumentation code will likely require an
update in docs. Please make sure to update the opentelemetry.io repo with any
docs changes.

A Pull Request that modifies docker-compose.yaml, otelcol-config.yaml, or
Grafana dashboards, will likely require an update to the Demo Helm chart.
Other changes affecting how a service is deployed will also likely require an
update to the Demo Helm chart.
-->

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

eliasmueller committed:
feat: ES-9 add EDMM Model (K8s) of otel-shop

MarcelWolkober commented on edmm/otel_store_k8s_translated.yaml:
```diff
@@ -0,0 +1,2678 @@
+      description: null
+      properties:
+        - cpu_count: 4
+        - ram_GiB: 16
```
Gibt's da einen "Fail-save" falls die 16 GB ram nicht komplett verfügbar sind? Oder beschreibt das nur das was der maximal haben darf?

eliasmueller replied:
letzteres denke ich, gibt sonst halt nen Fehler

eliasmueller committed:
feat: ES-9 add some refinement and connects_to

Arikuma97 committed:
Update otel_store_k8s_translated.yaml

YAML style adjusted to a uniform format.

Arikuma97 committed:
Update otel_store_k8s_translated.yaml

YAML style adjusted to a uniform format. Adjusted, of relation_types and components_types (underscore was missing) --> Inconsistency

eliasmueller committed:
Merge remote-tracking branch 'origin/demaf' into ES-9-2

eliasmueller committed:
Merge branch 'refs/heads/demaf' into ES-9-2
